### PR TITLE
chore: cache metamask version

### LINKED ISSUE #244: Cache Metamask in CI to avoid rate limit
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by mpetrunic**
**Short description of work done**

### PR Checklist

- [ ] I have run linter locally
- [ ] I have run unit and integration tests locally
  - [ ] Update configuration the newest version (readme and const)
- [ ] Rebased to master branch / merged master

### Changes

- _EXAMPLE - Removed basic auth from the controller_

### Issues

Closes #244

### CODE REVIEW

File: src/setup/utils/metaMaskDownloader.ts [hunk 1]
```diff
@@ -6,7 +6,13 @@ import * as path from "path";
...
 import StreamZip from "node-stream-zip";
 
-const defaultDirectory = path.resolve(__dirname, "..", "..", "..", ".metamask");
+export const defaultDirectory = path.resolve(
+  __dirname,
+  "..",
+  "..",
+  "..",
+  ".metamask"
+);
```
Comment (BeroBurny) [inline src/setup/utils/metaMaskDownloader.ts:15]: Think I did mess up this long ago.
Is it not better to get rid of `__dirname`?
```suggestion
export const defaultDirectory = path.resolve("./.metamask");
```

File: scripts/download_metamask.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+(async function() {
+    await downloader(RECOMMENDED_METAMASK_VERSION, {
+        location: defaultDirectory,
+        flask: false,
+      });
+    await downloader(RECOMMENDED_METAMASK_VERSION, {
+        location: defaultDirectory,
+        flask: true,
+      });
```
Comment (BeroBurny) [inline scripts/download_metamask.ts:12]: `location` is an optional param, seen a change in `src/setup/utils/metaMaskDownloader.ts` on `defaultDirectory` which is used here. Do not see a reason for it as not providing will use it.

File: src/setup/utils/metaMaskDownloader.ts (hunk 1 above)
Comment (mpetrunic) [reply src/setup/utils/metaMaskDownloader.ts:15]: Yes, but its a breaking change now 😅

Comment (BeroBurny) [reply src/setup/utils/metaMaskDownloader.ts:15]: using `.cache` sounds like a great idea, will keep files away from git, but will be pretty accessible

Comment (BeroBurny) [reply src/setup/utils/metaMaskDownloader.ts:15]: using `.cache` sounds like a great idea, will keep files away from git, but will be pretty accessible

Comment (mpetrunic) [reply src/setup/utils/metaMaskDownloader.ts:15]: True, and currently, users are storing metamask inside node_modules, inside dappeteer directory 😅

**Review by irubido — APPROVED** [review]

File: src/setup/utils/metaMaskDownloader.ts (hunk 1 above)
Comment (mpetrunic) [reply src/setup/utils/metaMaskDownloader.ts:15]: But I didn't change anything so there is nothing to revert 😅 but issue about .cache sounds good to me^^

**Review by BeroBurny — APPROVED** [review]

**Pull request merged**